app/views/steps/abuse_concerns/start/show.en.html.erb

```diff
@@ -3,16 +3,17 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <%= step_header %>
-
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">You and your children’s safety</h1>
+    <p class="app__section_heading">Safety concerns</p>
+    <h1 class="heading-xlarge gv-u-heading-xxlarge">You and your children</h1>
 
     <div class="govuk-govspeak gv-s-prose">
-      <p>Based on your previous answer, we’ll need to ask you some questions about your concerns.</p>
-      <p>The aim of the following questions is to provide more information for an advisor from the
-        <a href="https://www.cafcass.gov.uk/" target="external_link">Children and Family Court Advisory and Support
-          Service (Cafcass)</a>. They will carry out background checks before the first hearing.</p>
-      <p>Due to the sensitive nature of the questions you may find some of them difficult to answer.
-        If you don’t know an answer you can select ‘Don’t know’.</p>
+      <p>You told us:</p>
```

Comment:
This copy, in the prototype, is dynamic based on the answers to previous questions, meaning it will not always be "you think your children are affected by being in contact with someone who may have a drug, alcohol or substance problem".

But for the rails app, unless we have time left to elaborate these dynamic steps, we are going to leave it as generic static copy.

I proposed to replace the 'You told us:' block, with 'Based on your previous answer, we’ll need to ask you some questions about your concerns.' to be generic enough for now (there is already a paragraph with similar copy in this step already, so just modifying it a bit would be enough).